### PR TITLE
[CSS2] define <color> by reference to CSS Color 3

### PR DESCRIPTION
**Pull request by gsnedders**
Fixes https://github.com/w3c/csswg-drafts/issues/628.

See also: https://github.com/w3c/csswg-drafts/issues/5114.

This is in many ways me experimenting with what the working group actually thinks of some of the proposals in both those issues. :smile:

### CODE REVIEW

**Comment by frivoal** [comment]
Overall, this seems fine to me. Css2 should be updated to say the same thing as css-color 3, as #628 argues, and that's certainly the easiest way to do that.

A minor note: SVG11 is an informative reference of CSS2, but a normative reference of CSS-COLOR3. As it is itself a REC, I don't believe that's an issue.

So, from a normative point of view, I think there's no issue with this, and I support it.

From a patent policy point of view, the effect of doing this is not necessarily identical to adjusting the css2 text to mean the same as the level 3 text and republishing that.
Currently, we have patent commitment:
- on the css2.1 content, by the members who were there then
- on the css-color-3 content, by the members who were there then
Eventually we will also have patent commitment:
- on the css-color-4, which is a superset of 3, by the members who've involved in its development

If we do what you suggest, that's all we get.

If we publish a new CSS2.x REC with the level-2 content corrected to match level 3, we'd also get patent commitment on the corrected level 2 material from members involved during that time frame. I don't think there would be any member who'd end up committed to this adjusted level 2 who's not already committed to level4, and I'm not convinced we'd even get there sooner.

So, even considering the Patent Policy implications, I still think this is a good move.

File: css2/Overview.bs [hunk 1]
```diff
@@ -2684,142 +2692,16 @@ p:before {content: counter(par-num, upper-roman) ". "}
...
-em { color: rgb(110%, 0%, 0%) }  /* clipped to rgb(100%,0%,0%) */
-</pre>
-</div>
-
-<p>Other devices, such as printers, have different gamuts than sRGB;
-some colors outside the 0..255 sRGB range will be representable
-(inside the device gamut), while other colors inside the 0..255 sRGB
-range will be outside the device gamut and will thus be mapped.
-</p>
-<div class="note"><p><em><strong>Note.</strong> Mapping or clipping of
```
Comment (svgeesus) [inline css2/Overview.bs:2819]: cringing at the wooliness of that paragraph, good to have dropped it

**Comment by gsnedders** [comment]
I guess we probably want to get a WG resolution on this given this both includes substantive changes and patent commitments.

**Comment by gsnedders** [comment]
(FWIW, I'm not gonna be on the call this week, hopefully will be by next week; @svgeesus can you handle the above?)

**Comment by fantasai** [comment]
I don't think we should take this change. I think we should definitely add a note pointing to css-color-3, but I don't think we should be removing the entire section. 

I also would like to see CSS2 republished cleanly, and then changes made on top of that, rather than piling up a bunch of changes in the ED without much clarity wrt what's happening on /TR?

**Comment by svgeesus** [comment]
@fantasai could you explain why obsolete material should not be removed?

**Comment by svgeesus** [comment]
related https://github.com/w3c/csswg-drafts/issues/5114

**Comment by gsnedders** [comment]
**RESOLVED**: sections entirely replaced by RECs are marked with obsoletion notices recommending the REC, but not removed

**Pull request closed without merging**